templates/iptables.rules.tmpl

```diff
@@ -3,14 +3,14 @@
 :INPUT ACCEPT [0:0]
 :OUTPUT ACCEPT [0:0]
 :POSTROUTING ACCEPT [0:0]
--A POSTROUTING -s {{.DockerSubnet}} ! -d {{.DockerSubnet}} -j MASQUERADE
+{{if .UseDockerRules}}-A POSTROUTING -s {{.DockerSubnet}} ! -d {{.DockerSubnet}} -j MASQUERADE{{end}}
 COMMIT
 
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
 :OUTPUT DROP [0:0]
-:DOCKER - [0:0]
+{{if .UseDockerRules}}:DOCKER - [0:0]
```

Comment:
I like more this way because I can clearly see the docker rules as a block and not as multiple `if ...end`. That's why I chose a single `if else end` block`